DSL/conditions.md

```diff
@@ -0,0 +1,288 @@
+# Declarative DSL to Check Conditions in JSON Schema
+
+This document provides an overview of a declarative Domain Specific Language (DSL) designed to check various conditions in a JSON schema. The DSL is structured as an array of conditions, with each condition containing three mandatory fields: `path`, `operation`, and `value`.
+
+## DSL Structure
+
+### Condition Fields
+
+- **path**: Specifies the path to the target in the schema where the operation will be performed.
```

Comment:
Please use back ticks for inline code, instead of **, i.e.

```suggestion
- `path`: Specifies the path to the target in the schema where the operation will be performed.
```

Reply:
You should define the type of these properties. Right now the document is very vague. You should also mention why we don't use JSON Pointers here: for simplicity and to support the "pattern" extension we came up with